### PR TITLE
feat(ui): Add project series to session histogram

### PR DESCRIPTION
**Pull request by matejminar**
## Before
Screenshot 2021-03-19 at 17 35 01

## After
Screenshot 2021-03-19 at 17 34 24

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx [hunk 1]
```diff
@@ -282,56 +332,85 @@ class ReleaseHealthRequest extends React.Component<Props, State> {
...
+    const {statusCountByReleaseInPeriod, statusCountByProjectInPeriod} = this.state;
     const field = this.displayToField(display);
+
+    const intervals = statusCountByProjectInPeriod?.intervals ?? [];
+
+    const projectData = statusCountByProjectInPeriod?.groups
+      .filter(({by}) => by.project === project)
+      ?.reduce((acc, group) => {
+        group.series[field].forEach(
+          (value, index) => (acc[index] = (acc[index] ?? 0) + value)
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx:386]: Is it useful to have a local variable with this function as it shows up a few times here.

File: src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx [hunk 2]
```diff
@@ -163,7 +179,11 @@ class ReleaseHealthRequest extends React.Component<Props, State> {
     }
   };
 
-  // ?query=release%3Ae2da495e7f7d9f6ac3ff271cf06698032ee8f65d+OR+release%3Aec1fa0d40afdf5539f80846022c385bea0d06b40&interval=1h&statsPeriod=14d&project=11276&environment=prod&field=sum(session)&groupBy=project&groupBy=release&groupBy=session.status
+  /**
+   * Used to calculate crash free rate, count histogram (This Release series), and crash count
+   *
+   * ?query=release%3Ae2da495e7f7d9f6ac3ff271cf06698032ee8f65d+OR+release%3Aec1fa0d40afdf5539f80846022c385bea0d06b40&interval=1h&statsPeriod=14d&project=11276&environment=prod&field=sum(session)&groupBy=project&groupBy=release&groupBy=session.status
```
[the remark this answers is not in the document]
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx]: Not really, it helped me while building the right queries but we can remove it now 👍

File: src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx]: yes ❤️

**Review by priscilawebdev — APPROVED** [review]
lgtm.. Thanks for creating the tests in typescript 🙏

File: src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx [hunk 3]
```diff
@@ -282,56 +340,75 @@ class ReleaseHealthRequest extends React.Component<Props, State> {
...
+      ({by}) => by.project === project
+    )?.series[field];
+
+    const releaseData = totalCountByReleaseIn24h?.groups.find(
+      ({by}) => by.project === project && by.release === version
+    )?.series[field];
+
     return [
+      {
+        seriesName: t('Total Project'),
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx:368]: According to Dora's PR https://github.com/getsentry/sentry/pull/24581/files Total should the last in this list 😉

Comment (matejminar) [reply src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx:368]: I made it this way to remain consistent with the histogram on the Issues page. We'll sync with Dora and I will consolidate it in all places depending on how we decide 👍

**Pull request merged**